README.md

```diff
@@ -79,7 +79,9 @@ curl -X POST http://localhost:4434/createservice \
             "type": "postgres",
             "name": "localtest",
             "username": "spinup",
-            "password": "spinup"
+            "password": "spinup",
+            "memory": "6000", //In Megabytes
```

Comment:
So this is 6GB, correct? was curious if someone is running it on say...a system with < 6GB, what would docker do in that case?

Reply:
right,will have to add error messages for edge cases like this

Reply:
Yeah, was just curious anyway. Since this is just an example and users can adjust based on their system...it should likely be fine.